include/libcyphal/runnable.hpp

```diff
@@ -6,24 +6,34 @@
 #ifndef LIBCYPHAL_RUNNABLE_HPP_INCLUDED
 #define LIBCYPHAL_RUNNABLE_HPP_INCLUDED
 
+#include "transport/errors.hpp"
 #include "types.hpp"
 
+#include <cetl/pf17/cetlpf.hpp>
+#include <cetl/unbounded_variant.hpp>
+
+#include <cstddef>
+
 namespace libcyphal
 {
 
 class IRunnable
 {
+    static constexpr std::size_t MaxErrorSize = sizeof(transport::AnyError);
```

Comment:
Currently this is only one error type here, but my idea is that it will be like following in the future:
```
std::max(sizeof(transport::AnyError), sizeof(XxxLayer::YyyError), ...)
```
when there will be other layers specific error types added.

Reply:
ok, I'll make it constant (so that current form of `AnyError` will fit).

>if we run over the limit we'll get a build error.
Yes, it will fail to build.